scripts/features.sh

```diff
@@ -51,13 +51,13 @@ packages() {
 	shift
 
 	# We only allow variable names, parentheses and the operators: & | !
-	if [ "$(expr match "$cond" '.*[^A-Za-z0-9_()&|! ].*')" -gt 0 ]; then
+	if grep -q ".*[^A-Za-z0-9_()&|! ].*" <<< "$cond"; then
```

Comment:
- The leading and trailing `.*` are redundant
- Use single quotes for the regex